powerlevel9k.zsh-theme

```diff
@@ -909,10 +909,13 @@ prompt_dir() {
     "HOME"            "HOME_ICON"
     "HOME_SUBFOLDER"  "HOME_SUB_ICON"
     "NOT_WRITABLE"    "LOCK_ICON"
+    "ETC"             "ETC_ICON"
   )
   local state_path="$(print -P '%~')"
   local current_state="DEFAULT"
-  if [[ "${POWERLEVEL9K_DIR_SHOW_WRITABLE}" == true && ! -w "$PWD" ]]; then
+  if [[ $state_path == '/etc'* ]]; then
```

Comment:
I wonder if an option to turn this off should be present?

Reply:
Well. You could override `ETC_ICON`..

Reply:
Sure, that's an option. I was going off of `DIR_SHOW_WRITABLE` but looking at the rest, there's not really a set precedent as I thought, as home behaves exactly like this, Sorry!